src/components/AddBook.js

```diff
@@ -20,15 +20,20 @@ function AddBook() {
     dispatch(addNewBook({
       title, author, category: 'Fiction', item_id: `item${Math.floor(Math.random() * 1000)}`,
     }));
+    setTitle('');
+    setAuthor('');
   };
 
   return (
     <>
-      <form onSubmit={handleSubmit}>
-        <input type="text" value={title} placeholder="Book Title" onChange={handletitleChange} />
-        <input type="text" value={author} placeholder="author" onChange={handleSelectChange} />
-        <AddBookBtn />
-      </form>
+      <div className="add-container">
```

Comment:
- [ ] Please make sure that this form has the same width as that in the book cards above.
### Your implementation
![image](https://github.com/manzitresor/book_store/assets/45460192/cacfb9a4-5412-42cc-b894-64e3dc96f473)
### Required
![image](https://github.com/manzitresor/book_store/assets/45460192/e90739be-0938-4d0e-a466-09b4c8085273)